### ISSUE TITLE
False positive parse error when `match` is used as a variable

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#2413]
This is a tracking bug. For original, refer to https://github.com/microsoft/pylance-release/issues/1911.

```python
import numpy as np
match = np.ones((10,10))
match[2:8,2:8] = 0
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.178, which I just published. It will also be included in the next release of pylance.
